sources/features/gsub.fea

```diff
@@ -0,0 +1,752 @@
+languagesystem DFLT dflt;
+languagesystem cyrl dflt;
+languagesystem cyrl SRB ;
+languagesystem grek dflt;
+languagesystem hebr dflt;
+languagesystem latn AZE ;
+languagesystem latn CRT ;
+languagesystem latn dflt;
+languagesystem latn FIN ;
+languagesystem latn ISM ;
+languagesystem latn LSM ;
+languagesystem latn NOR ;
+languagesystem latn NSM ;
+languagesystem latn SKS ;
+languagesystem latn SSM ;
+languagesystem latn SVE ;
+languagesystem latn TRK ;
+
+# #include "locl.fea"
+
+# #include "small.fea"
+
+# #include "case.fea"
+
+# #include "contextual.fea"
+
+# #ifndef NOSMALLCAPS
+# #include "sc.fea"
+# #endif
+
+# #include "dhlig.fea"
+
+# #include "num.fea"
+
+# #include "sscv.fea"
+
+# #ifdef SANS
+# #include "liga_sans.fea"
+# #else
+# #include "liga.fea"
+# #endif
+
+feature locl {
+
+ script cyrl;
+     language SRB  exclude_dflt;
+      lookup locl_cyrillic;
+
+ script latn;
+     language AZE  exclude_dflt;
+# #ifndef SANS
+      lookup locl_undo_fi;
+# #endif
+     language CRT  exclude_dflt;
+# #ifndef SANS
+      lookup locl_undo_fi;
+# #endif
+     language FIN  exclude_dflt;
+      lookup locl_sami;
+     language ISM  exclude_dflt;
+      lookup locl_sami;
+     language LSM  exclude_dflt;
+      lookup locl_sami;
+     language NOR  exclude_dflt;
+      lookup locl_sami;
+     language NSM  exclude_dflt;
+      lookup locl_sami;
+     language SKS  exclude_dflt;
+      lookup locl_sami;
+     language SSM  exclude_dflt;
+      lookup locl_sami;
+     language SVE  exclude_dflt;
+      lookup locl_sami;
+     language TRK  exclude_dflt;
+      lookup locl_sami;
+# #ifndef SANS
+      lookup locl_undo_fi;
+# #endif
+} locl;
+
+feature frac {
+
+ script DFLT;
+     language dflt ;
+      lookup frac_contextual;
+      lookup frac_unicodepoint;
+
+ script cyrl;
+     language dflt ;
+      lookup frac_contextual;
+      lookup frac_unicodepoint;
+
+ script grek;
+     language dflt ;
+      lookup frac_contextual;
+      lookup frac_unicodepoint;
+
+ script hebr;
+     language dflt ;
+      lookup frac_contextual;
+
+ script latn;
+     language dflt ;
+      lookup frac_contextual;
+      lookup frac_unicodepoint;
+     language AZE  exclude_dflt;
+      lookup frac_contextual;
+      lookup frac_unicodepoint;
+     language CRT  exclude_dflt;
+      lookup frac_contextual;
+      lookup frac_unicodepoint;
+     language TRK  exclude_dflt;
+      lookup frac_contextual;
+      lookup frac_unicodepoint;
+} frac;
+
+feature sups {
```

Comment:
Same here and many other features below.

Reply:
(will be) fixed.